code/modules/core_implant/cruciform/machinery/obelisk.dm

```diff
@@ -32,7 +32,15 @@ GLOBAL_LIST_EMPTY(all_obelisk)
 /obj/machinery/power/nt_obelisk/New()
 	..()
 	GLOB.all_obelisk |= src
-
+/*
+	var/area/my_area = get_area(src.loc)
+	if(my_area.name == "Deep Maintenance") //Shockingly this is how get area works
+		//In deepmaints we work way worse,
+		area_radius = max(3, area_radius - 1 - GLOB.chaos_level)
+		damage = max(5, damage - 10 - GLOB.chaos_level)
+		max_targets = max(1, max_targets - 1 - GLOB.chaos_level)
+		desc = "[desc] This one is being weaked by the forces in the area"
+*/
```

Comment:
Oh, probably not good to just... add a bunch of commented out code

Reply:
It was ran by churchies they didnt like the idea, keeping it in for refence/tweaking in a more dedicated pr is all